docs/IAMPERMISSIONS.md

```diff
@@ -0,0 +1,249 @@
+# Limiting IAM Permissions
+
+It is possible to define a role limiting IAM permissions when deploying to Amazon EKS or ECS.
```

Comment:
I don't think TestSys can be deployed to ECS.
```suggestion
It is possible to define a role limiting IAM permissions when deploying TestSys to Amazon EKS.
```

Reply:
Reworded slightly different. Let me know if that's more clear or if it needs further work. Thanks!